discovery-provider/src/app.py

```diff
@@ -629,6 +629,10 @@ def configure_celery(celery, test_config=None):
     redis_inst.delete(INDEX_REACTIONS_LOCK)
     redis_inst.delete(UPDATE_TRACK_IS_AVAILABLE_LOCK)
 
+    redis_inst.delete("index_user_bank_backfill_complete")
+    redis_inst.delete("index_rewards_manager_backfill_complete")
+    redis_inst.delete("index_spl_token_backfill_complete")
```

Comment:
Don't think we need to / want to delete these here - don't we want these bools to persist across container restarts?

Reply:
Hmm... not sure it matters as they will be persisted in the db anyways.